src/mscorlib/model.xml

```diff
@@ -2326,6 +2355,17 @@
       <Member MemberType="Property" Name="InvalidCultureName" />
       <Member MemberType="Property" Name="Message" />
     </Type>
+    <Type Name="System.Globalization.CultureTypes">
+      <Member MemberType="Field" Name="AllCultures" />
```

Comment:
I am not in sync with the latest decisions but did we decide to expose CultureType? I am still recommending not to expose it. i tis mainly used with CultureInfo.GetCultures which you didn't expose it here either. It would be good if we can avoid expose both the type and the API.